books-i-read.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My Curriculum Vitae</title>
+    <meta charset="utf-8">
+	<meta name="Description" content="Curriculum Vitate">
+	<meta name="keywords" content="Name, Address, Education, Hobbies">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+	    <article>
+	    	<h2>About The Books I Read.</h2>
+	    	<ul>
+	    		<li>
+	    			<article>
+	    				<h3>How to Win Friends and Influence People.</h3>

```

Comment:
You _could_ use an h1 here. Because <article> starts a new section. But the reality is often that SEO's want this to be a subheading. so this is fine